### PR TITLE
Fixed typos and reworked the content

### PR DESCRIPTION
**Pull request by svekars**
Added the following changes to the QuickStart Guide:

- Fixed the typos
- Added the Required packages section
- Improved readability

### CODE REVIEW

**Comment by akscram** [comment]
Could you please use the unix file format in your favorite editor? (E.g. in `vim` it is possible to fix by the next command `:set ff=unix` or in your `~/.vimrc`.)

File: docs/quickstart.rst [hunk 1]
```diff
@@ -1,114 +1,271 @@
...
 
+If you build the Managed Services Tooling ISO manually, you need to install
+a specific set of packages required for the ISO. The latest packages are used
+automatically.
 
-Build Installation ISO
-----------------------
+The Managed Services Tooling ISO must include the following packages:
+
+.. list-table:: **Required packages**
```
Comment (akscram) [inline docs/quickstart.rst:72]: This list of packages are automatically put into an ISO during the build process and they are installed into the target operating system during the automatic installation from an ISO. This packages should not be installed on the build host. So, the build host is needed only to build an ISO. This is the major feature of the build process itself to pack a necessary list of packages into an ISO to be able then install hosts from this ISO without the Internet connection (the same as the offline installation).

Comment (svekars) [reply docs/quickstart.rst:72]: Does the **building environment** need an Internet connection or a connection to a repo with the required packages?

Comment (akscram) [reply docs/quickstart.rst:72]: Yes, the building environment needs an Internet connection.

**Comment by akscram** [comment]
The commit message does not longer reflect changes in the following item:

`- Added the Required packages section`

**Pull request merged**